### ISSUE TITLE
Question - Is there a way to show a placeholder to add an item after deleting an Item? 

### ISSUE THREAD

**Issue by dinbtechit** [baoolong/DragableGridview#13]
Question - Is there a way to show a placeholder to add an item after deleting an Item? 

Any help is much appreciated.

**For example: (for a Grid size of 3 x 3)**

**Step 1** - Delete Position - 7

![image](https://user-images.githubusercontent.com/17984781/60912659-416f1280-a25c-11e9-9e6b-f41ed41dd3f3.png)

**Step 2** -  Item at position 8 swaps to position 7 (Already works that way)

![image](https://user-images.githubusercontent.com/17984781/60912985-cce8a380-a25c-11e9-937a-ab5c98a0c7f1.png)

**Step 3** - Item 8 now shows a placeholder to add new items.

![image](https://user-images.githubusercontent.com/17984781/60912788-8004cd00-a25c-11e9-93e2-0df9be657091.png)

**Comment by gzlock** [comment]
Can not add a new item at last index?

**Comment by dinbtechit** [comment]
> Can not add a new item at last index?

Yeah I tried doing that. Problem is, user can move around the placeholder. Is there by any chance can we lock a tile?

**Comment by gzlock** [comment]
> > Can not add a new item at last index?
> 
> Yeah I tried doing that. Problem is, user can move around the placeholder. Is there by any chance can we lock a tile?

![ezgif-1-0878d1997887](https://user-images.githubusercontent.com/2311740/67093118-7982db80-f1e3-11e9-831c-b57f336dab5d.gif)

I spent a few days to develop a widget that meet your requirements.

Each children is draggable, fixable and deletable.

There may be some minor issues, but I will maintain this project.

Hope you will like it.

https://pub.dev/packages/draggable_container

https://github.com/gzlock/flutter_draggable_container

**Comment by gzlock** [comment]
> > Can not add a new item at last index?
> 
> Yeah I tried doing that. Problem is, user can move around the placeholder. Is there by any chance can we lock a tile?

![ezgif-1-0878d1997887](https://user-images.githubusercontent.com/2311740/67093118-7982db80-f1e3-11e9-831c-b57f336dab5d.gif)

I spent a few days to develop a widget that meet your requirements.

Each children is draggable, fixable and deletable.

There may be some minor issues, but I will maintain this project.

Hope you will like it.

https://pub.dev/packages/draggable_container

https://github.com/gzlock/flutter_draggable_container

**Comment by dinbtechit** [comment]
That looks freaking awesome! Great job. I'm closing this ticket.

**Comment by dinbtechit** [comment]
This looks freaking awesome! great job. I'll try it out and let you know.

**Comment by gzlock** [comment]
> This looks freaking awesome! great job. I'll try it out and let you know.

Hope you can talk with me in the telegram  https://t.me/gzlock

**Comment by dinbtechit** [comment]
> > This looks freaking awesome! great job. I'll try it out and let you know.
> 
> Hope you can talk with me in the telegram https://t.me/gzlock

sure will do.

**Comment by gzlock** [comment]
> > > This looks freaking awesome! great job. I'll try it out and let you know.
> > 
> > 
> > Hope you can talk with me in the telegram https://t.me/gzlock
> 
> sure will do.

I want to discuss with you how to add a new item.I don't know how to do that when items include the fixed item.

**Comment by dinbtechit** [comment]
> > > > This looks freaking awesome! great job. I'll try it out and let you know.
> > > 
> > > 
> > > Hope you can talk with me in the telegram https://t.me/gzlock
> > 
> > 
> > sure will do.
> 
> I want to discuss with you how to add a new item.I don't know how to do that when items include the fixed item.

Sure we can discuss that. I also found few bugs. I'll contact you on telegram.
